server/models/db.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-console */
+import 'dotenv/config';
+import pg from 'pg';
+
+const { NODE_ENV } = process.env;
+const env = NODE_ENV === 'test' || NODE_ENV === 'dev' ? `_${NODE_ENV}`.toUpperCase() : '';
+
+const pool = new pg.Pool({
+  connectionString: process.env[`DATABASE_URL${env}`],
+});
+
+pool.on('connect', () => {
+  console.log('connected to the Database');
+});
+
+const dropTables = () => {
+  const usersTable = 'DROP TABLE IF EXISTS users;';
+  const entriesTable = 'DROP TABLE IF EXISTS entries;';
+
+  const dropTablesQueries = `${entriesTable};${usersTable}`;
+
+  pool
+    .query(dropTablesQueries)
+    .then((res) => {
+      console.log(res);

```

Comment:
Please, we do not push `development tools` into `production code`. so remove all console.log()